src/style.css

```diff
@@ -1 +1,85 @@
-/* Styles go here */
+html {
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 14px;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.header {
+  display: flex;
+  align-items: stretch;
```

Comment:
do you need this property? it was set by default, try to delete it and check, if anything will be changed